### DISCUSSION TITLE
No overlapping of events

### DISCUSSION THREAD

**Discussion by gekberlin** [MMRIZE/MMM-CalendarExt3Journal#10] [Q&A]
Hi, 
is there any possibility to adjust the css in the way that events that are scheduled at the same time are not displays overlayed over each other ? Something like the popular family boards where the events are displayed next to each other?

Now: 
![image](https://github.com/user-attachments/assets/e054d393-0290-4494-bffb-0e4ceea74385)

Wish:
![image](https://github.com/user-attachments/assets/4b5eb433-57e0-4538-b481-e189516be9b0)

**Comment by eouia** [comment]
This functionality has been under consideration for some time. While there is an experimental fork available, it has not yet been merged into the main version and thus has not been released.  

One issue is the increased complexity of the logic, but more importantly, there are concerns about its practicality due to potential readability issues. Narrower event display widths could negatively impact readability.  

In scenarios where multiple events occur during overlapping time slots, this type of visualization may not be well-suited to the characteristics of MagicMirror. Features like scrolling, zooming, or clicking to reveal details are limited, making it less practical. It might handle two overlapping events reasonably well, but once this kind of area division is allowed, there’s no way to prevent it from theoretically accommodating three or four overlapping events.  

If the display is limited to a single day instead of several days, it may be possible to secure enough cell width, even for three or four overlapping events.  

However, in such cases, the `MMM-CalendarExt3Timeline` module might be a more appropriate choice than this module.

**Reply by gekberlin** [reply]
Thanks for the fast reply @eouia ! 
I already thought so that my request could be a hard job with plain css. 
Any chance that I can have sneak preview on the experimental fork?
Best and happy new year!
